CONTRIBUTING.md

```diff
@@ -0,0 +1,94 @@
+# Contributing
```

Comment:
needs some editing -- as it's referring to old comm channels etc.

Reply:
changed the gitter link to discoard